languages/julia/exercises/concept/exercism-matrix/.docs/introduction.md

```diff
@@ -1,5 +1,3 @@
-## matrices-introduction
-
 ## matrices-concatenation
 
 ## matrices-indices
```

Comment:
Shouldn't these be removed too then?

Reply:
They should, but my script did not remove level 2 headings in general, just those that matched the exercise's concept. And those headings don't correspond to any of the exercise's concepts.